tests/unit/test_network_connections.py

```diff
@@ -437,6 +437,8 @@ def test_validate_dict(self):
             {"i": 5, "s": "s_default", "l": "6"}, v.validate({"i": "5", "l": "6"})
         )
         self.assertValidationError(v, {"k": 1})
+        # Assert that `None` is normalized into `{}`` in ArgvalidatorDict()
```

Comment:
This comment can be removed IMHO, since it describes the code instead of providing additional information.

Reply:
Removed